src/v2/view-builder/utils/FactorUtil.js

```diff
@@ -49,6 +49,12 @@ const factorData = {
     description: loc('oie.webauthn.description', 'login'),
     iconClassName: 'mfa-webauthn',
   },
+
+  'app': {
+    label: loc('oie.app', 'login'),
```

Comment:
should the label not be the displayName from https://github.com/okta/okta-signin-widget/pull/1291/files#diff-7b4f6eaf0714ce098b2871fa2b55df38R232

Reply:
This is good for m1. For m2 we need that. It is still undecided if the remediation label would have it or we need to get form authenticatorEnrollment.